tests/unit/withOnyxTest.js

```diff
@@ -68,9 +68,13 @@ describe('withOnyx', () => {
         })(ViewWithCollections);
         const onRender = jest.fn();
         render(<TestComponentWithOnyx onRender={onRender} />);
-
-        Onyx.mergeCollection(ONYX_KEYS.COLLECTION.TEST_KEY, {test_1: {ID: 123}, test_2: {ID: 234}, test_3: {ID: 345}});
-        return waitForPromisesToResolve()
+        waitForPromisesToResolve()
```

Comment:
I think these tests are fixed correctly. Did not observe any issues with `mergeCollection()` yet